apps/landing/README.md

```
@@ -0,0 +1,69 @@
+### Scripts
+
+| Command | Purpose |
+| --- | --- |
+| `pnpm dev` | Start the local dev server |
+| `pnpm build` | Production build |
+| `pnpm start` | Run the production build |
+| `pnpm lint` | Run `next lint` |
+
+## Project layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)